### DISCUSSION TITLE
Prevent IM adding title/creation date/Mod date... when converting from jpg to pdf

### DISCUSSION THREAD

**Discussion by atulatg** [ImageMagick/ImageMagick#6874] [Development] [answered]
### ImageMagick version

ImageMagick-7.1.0-5

### Operating system, version and so on

Windows

### Description

Hi,
I'm converting from JPG to PDF but got an issue like below:
After converting and open it with Chrome, the name of PDF file is weird with something like 
![image](https://github.com/ImageMagick/ImageMagick/assets/25297027/ced9c989-3ed3-43e8-88e5-8364f823945b)
I've checked inside the file and found that IM added a title obj 
> /Title <006A007000670066006F0072006D0061007400740065007300740000>

This additional obj make the issue with name of PDF file when opening with Chrome, make the name have weird data.
Is there anyway to let IM skip adding title when converting from JPG to PDF? Thank you.

**Comment by snibgo** [comment]
Please show the command you used, and your input JPG file.

**Reply by atulatg** [reply]
Hi @snibgo ,
here is my command:
`magick.exe convert jpgformattest.jpg jpgformattest.pdf`
And here is my sample file: 
![jpgformattest](https://github.com/ImageMagick/ImageMagick/assets/25297027/ee599e7d-ac85-4a46-8df5-d1a31cf1056e)
And here is converted file:
[jpgformattest.pdf](https://github.com/ImageMagick/ImageMagick/files/13407394/jpgformattest.pdf)

**Comment by snibgo** [comment, marked as answer]
It looks as if your program (Chrome?) is misreading UTF16 as UTF8.

You can set a title, or set no title, like this:
```
magick in.jpg -define pdf:title="my title" out1.pdf

magick in.jpg -define pdf:title="" out2.pdf
```

See https://imagemagick.org/script/defines.php for other pdf defines.
